build/manifest/main.go

```diff
@@ -62,6 +111,32 @@ func findManifest() (*model.Manifest, error) {
 		return nil, errors.Wrap(err, "failed to parse manifest")
 	}
 
+	// If no version is listed in the manifest, generate one based on the state of the current
+	// commit, and use the first version we find (to prevent causing errors)
+	if manifest.Version == "" {
```

Comment:
Should the documentation make it clear that plugin devs can choose to use git-based versions or file-based versions?

Reply:
Good idea! Updated README.md.